postcss.js

```diff
@@ -35,20 +35,14 @@ function parallelCallback(err, results) {
 // SEE: https://www.npmjs.com/package/postcss#js-api
 /**
  * Build styles for the Core CMS
- * @param {boolean} [shouldFreezeVariables=false] - Whether to freeze values of custom properties
  */
-function buildStylesCore({shouldFreezeVariables = false} = {}) {
+function buildStylesCore() {
   let command;
   let sourceDir;
   let configDir;
 
-  if (shouldFreezeVariables === true) {
-    sourceDir = 'freeze_variables/';
-    configDir = 'conf/css/freeze_variables/';
-  } else {
-    sourceDir = '';
-    configDir = standardConfigDir;
-  }
+  sourceDir = '';
+  configDir = standardConfigDir;
```

Comment:
**Explanation.** Remove "freeze variables" hack that only existed to support variables (borrowed from Portal) to support search bar styles (borrowed from portal) (because it had no official UI at the time of implementation).